src/Native/Runtime/PalRedhawk.h

```diff
@@ -788,6 +788,10 @@ REDHAWK_PALIMPORT UInt32_BOOL REDHAWK_PALAPI PalAllocateThunksFromTemplate(_In_
 
 REDHAWK_PALIMPORT UInt32 REDHAWK_PALAPI PalCompatibleWaitAny(UInt32_BOOL alertable, UInt32 timeout, UInt32 count, HANDLE* pHandles, UInt32_BOOL allowReentrantWait);
 
+REDHAWK_PALIMPORT bool REDHAWK_PALAPI PalRegisterThreadShutdownCallback(ThreadShutdownCallback callback);
```

Comment:
I do not look at this PAL as independent piece. I think it would be ok to just call extern C method with given name from the PAL, and not have this registration.